docs/lts_policy.rst

```diff
@@ -0,0 +1,17 @@
+*******************
+LTS Backport Policy
+*******************
+
+Starting from astropy 5.0, backports to Long-Term Stable (LTS) releases
+will only cover:
```

Comment:
After the standard support life cycle.

Reply:
@Cadair , can you please elaborate on this?

Reply:
Well why the LTS version is both "the LTS" and the latest release the backport policy is the same as normal?